data/data/gcp/network/lb-private.tf

```diff
@@ -7,7 +7,18 @@ resource "google_compute_address" "cluster_ip" {
 resource "google_compute_health_check" "api_internal" {
   name = "${var.cluster_id}-api-internal"
 
+  // CAUTION: the gcp-routes mechanism must be _faster_ than this value:
+  //
+  //          Otherwise, local client traffic will go to the GCP LB until the
+  //          gcp-routes mechanism is done rerouting, and therefore back to the
+  //          local node in 1/3 of cases, which is blackholed (due to missing
+  //          hairpinning support).
   healthy_threshold   = 3
+  // CAUTION: the gcp-routes mechanism must be _slower_ than this value:
```

Comment:
Hrm.  If both `healthy_threshold` and `unhealthy_threshold` are 3, how can the former be faster than the gcp-routes mechanism while the latter is slower than that same gcp-routes mechanism?

Reply:
You assume that the gcp-routes mechanism is symmetric. It isn't: 

  https://github.com/openshift/machine-config-operator/pull/1670/files#diff-70c7d95d80d3256f98f75f82f7e63b00R111

Reply:
So in theory:
- 3*2s (LB noticing green readyz) > 1*2s (gcp-routes noticing green readyz)
- 3*2s (LB noticing red readyz) < 8*2s (gcp-routes noticing red readyz)

Reply:
But this might be an issue, cc @squeed:

https://github.com/openshift/machine-config-operator/pull/1670/files#diff-c8e870b664ab100f0b50218e3fdd5d46R143

gcp-routes only polls every 5s. So `1*2s` could become `1*2s+5s` which is bigger than `3*2s`.